ajax/libs/open-color/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "open-color",
+  "filename": "open-color.min.css",
+  "version": "1.5.1",
+  "description": "Open color scheme for web UI",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/yeun/open-color.git"
+  },
+  "keywords": [
+    "color",
+    "colorscheme",
+    "color swatches",
+    "CSS"
+  ],
+  "author": "Jeong Heeyeun <hyd5012@gmail.com>",
+  "homepage": "https://yeun.github.io/open-color/",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/yeun/open-color.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "open-color.!(json|less|scss|styl)"
```

Comment:
Let's use included pattern in BasePath `open-color*.+(js|css|map)`
As usual, we use included mode in root or `src` dir to add the files we need, and use excluded mode in `dist`, `build`... to exclude some file we don't need.
Because we treat `dist` as trust folder, root or `src` as not trust folder to add all the files.
Thanks.